src/ServiceClient.js

```diff
@@ -0,0 +1,184 @@
+import grpc, { InterceptingCall } from "grpc";
+import url from "url";
+import { BigNumber } from 'bignumber.js';
+import { find, map } from 'lodash';
+import paymentChannelStateServices from './payment_channel_state_service_grpc_pb';
+
+export default class ServiceClient {
+  constructor(sdk, metadata, group, ServiceStub, paymentChannelManagementStrategy, options = {}) {
+    this._sdk = sdk;
+    this._options = options;
+    this._metadata = metadata;
+    this._group = group;
+    this._paymentChannelManagementStrategy = paymentChannelManagementStrategy;
+    this._grpcStub = this._generateGrpcStub(ServiceStub);
+    this._paymentChannelStateServiceClient = this._generatePaymentChannelStateServiceClient();
+    this._paymentChannels = [];
+  }
+
+  get stub() {
+    return this._grpcStub;
+  }
+
+  get group() {
+    return this._group;
+  }
+
+  get groupIdInBytes() {
```

Comment:
This method could be a part of group instance which is returned by `group()` method.

Reply:
Agree. Addressed in [commit](https://github.com/singnet/snet-sdk-js/pull/1/commits/78cf9ae1c7ce7ce7b1812ea3d016dc3691663314)